data/core/units/elves/Druid.cfg

```diff
@@ -16,7 +16,7 @@
     advances_to=Elvish Shyde
     cost=25
     usage=healer
-    description= _ "Some elves choose to deepen their connection to the corporeal world, thereby enhancing their already potent abilities to foster and manipulate natural energies. Perhaps the most notable effect of these druids’ magic is the marked sense of motion in the deep woodlands that they preserve. Outsiders wandering through Elven woods often speak of entangling roots and vines, moving nearly as fast as a warrior’s marching pace, or the creaking shamble of ancient trees, lumbering across entire camps in just a single night. The forest and the earth become almost like a connected entity, something that lives and breathes like a single massive organism. As it flourishes under the druids’ careful tending, the forest kindles their magics in turn, symbiotically enhancing the already potent powers that these elves possess."
+    description= _ "Some elves choose to deepen their connection to the corporeal world, thereby enhancing their already potent abilities to foster and manipulate natural energies. Perhaps the most notable effect of these Druids’ magic is the marked sense of motion in the deep woodlands that they preserve. Outsiders wandering through Elven woods often speak of entangling roots and vines, moving nearly as fast as a warrior’s marching pace, or the creaking shamble of ancient trees, lumbering across entire camps in just a single night. The forest and the earth become almost like a connected entity, something that lives and breathes like a single massive organism. As it flourishes under the druids’ careful tending, the forest kindles their magics in turn, symbiotically enhancing the already potent powers that these elves possess."
```

Comment:
I know it's there in the original but I think 'Elven woods' should be 'elven woods'.